package.json

```diff
@@ -83,13 +83,16 @@
     "mkdirp": "^0.5.1",
     "nconf": "^0.8.4",
     "node-fetch": "^1.7.1",
+    "nodegit": "^0.20.2",
     "opn": "^5.1.0",
     "ora": "^1.2.0",
     "progress": "^2.0.0",
     "tableschema": "1.0.0-alpha.14",
+    "tmp": "0.0.33",
     "tv4": "^1.3.0",
     "url-join": "^2.0.2",
-    "xlsx": "^0.10.8"
+    "xlsx": "^0.10.8",
+    "yamljs": "^0.3.0"
```

Comment:
Can we add new dependencies using yarn add xxx and making sure they are in alphabetical order 😄